tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2379,14 +2465,33 @@ public static String simplify(String exampleHtml) {
     }
 
     public static String simplify(String exampleHtml, boolean internal) {
-        return exampleHtml == null ? null
-            : internal ? "〖" + exampleHtml
-                .replace("", "❬")
-            .replace("", "❭") + "〗"
-            : exampleHtml
-            .replace("<div class='cldr_example'>", "〖")
-            .replace("</div>", "〗")
-            .replace("<span class='cldr_substituted'>", "❬")
-            .replace("</span>", "❭");
+        if (exampleHtml == null) {
+            return null;
+        }
+        if (internal) {
+            return "〖"
+                    + exampleHtml
+                        .replace("", "❬")
+                        .replace("", "❭")
```

Comment:
string literals with PUA characters make this code obscure and fragile... not your fault since the code was already like this... still it would be better to use backgroundStartSymbol (already defined) instead of "", and so forth